src/logger/src/lib.rs

```diff
@@ -547,6 +547,7 @@ impl Logger {
                     METRICS.logger.missed_log_count.inc();
                 }
             } else {
+                METRICS.logger.missed_log_count.inc();
                 panic!("Failed to write to the provided metrics destination due to poisoned lock");
```

Comment:
I think here should be `Failed to write to the provided log destination due to...`, it is a miss of some previous commit.

Reply:
yes @dianpopa

Reply:
done 👍